<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InletPropDialog</class>
 <widget class="QDialog" name="InletPropDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit inlet properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0" columnstretch="60,0,0,0">
   <item row="13" column="2">
    <widget class="QLabel" name="labelSectorSize">
     <property name="text">
      <string>Angular sector size:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="14" column="2" colspan="2">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="3">
    <widget class="QSpinBox" name="spinCWAngle">
     <property name="suffix">
      <string>째</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
    </widget>
   </item>
   <item row="10" column="3">
    <widget class="QCheckBox" name="checkCCW">
     <property name="text">
      <string>CCW orientation</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="spinInnerRadius">
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> pixel</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000000000</number>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLabel" name="labelCCWAngle">
     <property name="text">
      <string>Angle boundary (CCW):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="12" column="3">
    <widget class="QSpinBox" name="spinSectors">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="2" colspan="2">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="labelOuterRadius">
     <property name="text">
      <string>Outer radius:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Inner radius:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="16" column="2" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="labelPosition">
     <property name="text">
      <string>%d, %d</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="13" column="3">
    <widget class="QSpinBox" name="spinSectorSize">
     <property name="suffix">
      <string>째</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>90</number>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QSpinBox" name="spinCCWAngle">
     <property name="suffix">
      <string>째</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="15" column="2" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="2">
    <widget class="QLabel" name="labelRefAngle">
     <property name="text">
      <string>Reference angle:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QLabel" name="labelCWAngle">
     <property name="text">
      <string>Angle boundary (CW):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QSpinBox" name="spinRefAngle">
     <property name="suffix">
      <string>째</string>
     </property>
     <property name="maximum">
      <number>359</number>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QLabel" name="labelSectors">
     <property name="text">
      <string>Sectors along radius:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Center:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QSpinBox" name="spinOuterRadius">
     <property name="suffix">
      <string> pixel</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000000000</number>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" rowspan="17">
    <widget class="QGraphicsView" name="imageView">
     <property name="interactive">
      <bool>false</bool>
     </property>
     <property name="renderHints">
      <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <widget class="QCheckBox" name="checkMainInlet">
     <property name="text">
      <string>Main inlet (origin)</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>labelSectorSize</zorder>
  <zorder>spinCWAngle</zorder>
  <zorder>checkCCW</zorder>
  <zorder>labelCCWAngle</zorder>
  <zorder>spinSectors</zorder>
  <zorder>labelOuterRadius</zorder>
  <zorder>label_2</zorder>
  <zorder>buttonBox</zorder>
  <zorder>labelPosition</zorder>
  <zorder>spinSectorSize</zorder>
  <zorder>spinCCWAngle</zorder>
  <zorder>labelRefAngle</zorder>
  <zorder>labelCWAngle</zorder>
  <zorder>spinRefAngle</zorder>
  <zorder>labelSectors</zorder>
  <zorder>label</zorder>
  <zorder>spinOuterRadius</zorder>
  <zorder>imageView</zorder>
  <zorder>checkMainInlet</zorder>
  <zorder>spinInnerRadius</zorder>
 </widget>
 <tabstops>
  <tabstop>spinInnerRadius</tabstop>
  <tabstop>checkMainInlet</tabstop>
  <tabstop>spinOuterRadius</tabstop>
  <tabstop>spinRefAngle</tabstop>
  <tabstop>spinCCWAngle</tabstop>
  <tabstop>spinCWAngle</tabstop>
  <tabstop>checkCCW</tabstop>
  <tabstop>spinSectors</tabstop>
  <tabstop>spinSectorSize</tabstop>
  <tabstop>imageView</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>InletPropDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>827</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InletPropDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>839</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
